src/common.js

```diff
@@ -0,0 +1,37 @@
+/** Shared logic */
+export const ecoindexUrl = 'https://www.ecoindex.fr';
+export const apiUrl = 'https://bff.ecoindex.fr/api';
+
+export const FETCH_RESULT_URL = (url, refresh = false) => `${apiUrl}/results/?url=${url}&refresh=${refresh}`;
+export const FETCH_TASK_URL = `${apiUrl}/tasks`;
+export const FETCH_ID_TASK_URL = (id) => `${FETCH_TASK_URL}/${id}`;
+export const FETCH_SCREENSHOT_URL = (id) => `${apiUrl}/screenshot/${id}`;
+export const FETCH_RESULT_ECOINDEX_URL = (id) => `${ecoindexUrl}/resultat/?id=${id}`;
+export const defaultColor = '#5b5b5b';
```

Comment:
Maybe defaultColor should be uppercase too as this is a global const...

Reply:
Yes, I add also DEFAULT_BADGE_TEXT